modin/pandas/dataframe.py

```diff
@@ -2720,6 +2721,23 @@ def __delitem__(self, key):
             raise KeyError(key)
         self._update_inplace(new_query_compiler=self._query_compiler.delitem(key))
 
+    @_doc_binary_op(
+        operation="integer division and modulo",
+        bin_op="divmod",
+        returns="tuple of two DataFrames",
+    )
```

Comment:
Why should we add this instead of a single-line docstring as we do for all other methods?

Reply:
Why not? I reused how Series did it.

Reply:
Ok, let's leave as is for now but we should probably re-consider the approach we will follow with.